streams/src/test/java/org/apache/kafka/streams/kstream/WindowsTest.java

```diff
@@ -58,19 +65,6 @@ public void shouldSetWindowRetentionTime() {
         assertEquals(anyNotNegativeRetentionTime, new TestWindows().until(anyNotNegativeRetentionTime).maintainMs());
     }
 
-
-    @Test
-    public void gracePeriodShouldEnforceBoundaries() {
```

Comment:
Similar to `UnlimitedWindows`, `TestWindows` just doesn't need to provide an API to set the grace period.